dev-tools/prev-gen-release-notes-settings.json

```
@@ -0,0 +1,60 @@
+{
+    "$schema": "https://raw.githubusercontent.com/KinsonDigital/kd-admin/preview/schemas/gen-release-notes-schema.json",
+    "ownerName": "KinsonDigital",
+    "repoName": "Infrastructure",
+    "githubTokenEnvVarName": "CICD_TOKEN",
+    "milestoneName": "${VERSION}",
+    "headerText": "${REPONAME} ${RELEASETYPE} Release Notes - ${VERSION}",
+    "releaseType": "Preview",
+    "extraInfo": {
+        "title": "Quick Reminder",
+        "text": "As with all software, there is always a chance for issues and bugs, especially for preview releases, so your input is greatly appreciated. 🙏🏼"
+    },
+    "emojisToRemoveFromTitle": [
+        "🚧",
+        "🐛",
+        "✨",
+        "🚀",
+        "🧪",
+        "🔬"
+    ],
+    "issueCategoryLabelMappings": {
+        "New Features ✨": "new-feature",
```

Fix new feature label name:

```suggestion
        "New Features ✨": "✨new-feature",
```